### DISCUSSION TITLE
Get reactive instead of ref?

### DISCUSSION THREAD

**Discussion by Extarys** [wobsoriano/trpc-nuxt#103] [Q&A] [answered]
I'm using `tabulator-tables` and it works better when using a `reactive` for the data. Is there any way to return a reactive from `useQuery` instead of a `ref`?

Thanks

**Comment by wobsoriano** [comment, marked as answer]
`useQuery` is built on top of `useAsyncData` and will always return an object with ref as values.

**Reply by Extarys** [reply]
Thank you, got it. I'll convert it with vueuse in the meantime while I'm looking for a more elegant solution - or until tabulator works better with ref :)

**Reply by wobsoriano** [reply]
FYI there’s toReactive function from vueuse that accepts a ref

**Reply by Extarys** [reply]
Yes that is what I'm gonna use for now, just not sure how much this impact performance and memory usage. But for now it'll do :D
